src/stores/businessBootstrap.ts

```diff
@@ -32,12 +36,91 @@ export const useBcrosBusinessBootstrap = defineStore('bcros/businessBootstrap',
     return `${GetCorpFullDescription(bootstrapLegalType.value)}${extraDesc}${filingName}`
   })
 
+  const isAmalgamationTodo = computed(() =>
+    bootstrapFilingType.value === FilingTypes.AMALGAMATION_APPLICATION &&
+    (
+      bootstrapFilingStatus.value === FilingStatusE.DRAFT ||
+      bootstrapFilingStatus.value === FilingStatusE.NEW
+    )
+  )
+  const isAmalgamationFiling = computed(() =>
+    bootstrapFilingType.value === FilingTypes.AMALGAMATION_APPLICATION &&
+    (
+      bootstrapFilingStatus.value === FilingStatusE.CANCELLED ||
+      bootstrapFilingStatus.value === FilingStatusE.PAID
+    )
+  )
+  const isContinuationInTodo = computed(() =>
+    bootstrapFilingType.value === FilingTypes.CONTINUATION_IN &&
+    (
+      bootstrapFilingStatus.value === FilingStatusE.CHANGE_REQUESTED ||
```

Comment:
I think you could have checked this with by creating array of valid enum values, and asking if the array includes specific value, it would be a bit more easy on read, but this is good too

Reply:
That is a good point. I will update it before merging